chainer/functions/array/depth2space.py

```diff
@@ -23,23 +23,19 @@ def forward(self, inputs):
         xp = cuda.get_array_module(X)
         bsize, c, a, b = X.shape
         c //= self.r ** 2
-        X = xp.transpose(X, (0, 2, 3, 1))
```

Comment:
It seems the original code is better for CPU.
Could you separate the code? (using `if xp is numpy:`)

Reply:
Thanks for review and performance evaluation.
I evaluated on CPU too. I confirmed performance decreased.
I will change the code with your suggestions.